src/fu-util.c

```diff
@@ -4865,6 +4955,22 @@ main(int argc, char *argv[])
 			      /* TRANSLATORS: command description */
 			      _("Inhibit the system to prevent upgrades"),
 			      fu_util_inhibit);
+
+	fu_util_cmd_array_add(cmd_array,
+			      "repair",
+			      /* TRANSLATORS: command argument: uppercase, spaces->dashes */
+			      _("[STREAM_ID]"),
+			      /* TRANSLATORS: command description */
+			      _("Automatically repair the system configuration to improve host security"),
+			      fu_util_auto_repair);
+
+	fu_util_cmd_array_add(cmd_array,
+			      "repair-list",
+			      NULL,
+			      /* TRANSLATORS: command description */
+			      _("List auto-repairing items"),
+			      fu_util_repair_list_items);
+
```

Comment:
TBH, I don't see a lot of value in this command.  Can you explain why it would be used?

Reply:
If it's only for bash completion it should be a list of appstream ID's not pretty descriptions.

Reply:
repair-list will list the appstream ID for the next revision.